NewsApp/NewsApp/Controller/ProfileController.swift

```diff
@@ -80,13 +87,23 @@ extension ProfileController: UIPickerViewDataSource {
 }
 
 extension ProfileController: UIPickerViewDelegate {
-    func pickerView(_ pickerView: UIPickerView, titleForRow row: Int, forComponent component: Int) -> String? {
-        return Country.allCases[row].name
+//    Pour utiliser sans FlagKit, décommenter cette fonction et supprimer func pickerView(viewForRow)
+
+//    func pickerView(_ pickerView: UIPickerView, titleForRow row: Int, forComponent component: Int) -> String? {
+//        return Country.allCases[row].name
+//    }
```

Comment:
D'une manière général, il ne fait pas push de code commentée, il y a des exception bien evidemment ;-)

Reply:
fix in "correction PR"